pydantic/networks.py

```diff
@@ -380,7 +382,7 @@ def validate(cls, value: NetworkType) -> Union[IPv4Network, IPv6Network]:
 pretty_email_regex = re.compile(r'([\w ]*?) *<(.*)> *')
 
 
-def validate_email(value: str) -> Tuple[str, str]:
+def validate_email(value: Union[str]) -> Tuple[Union[str], Union[str]]:
```

Comment:
```suggestion
def validate_email(value: Union[str]) -> Tuple[str, str]:
```
I think should be fine?

Reply:
@samuelcolvin 
I'm sorry, I misunderstood.